scripts/extensions/sams/src/components/assets/assetListItem.tsx

```diff
@@ -72,7 +72,9 @@ export class AssetListItem extends React.PureComponent<IProps> {
             <ListItem
                 onClick={this.onItemClick}
                 onDoubleClick={this.onItemDoubleClick}
-                selected={this.props.selected || this.props.itemSelected} shadow={1}>
+                selected={this.props.selected || this.props.itemSelected}
+                shadow={1}
+            >
                 <ListItemBorder />
```

Comment:
This component has a `state` property that we can set to `locked` if this Asset is locked (i.e. has `lock_user`, `lock_session`, set). This might be handy to have `isItemLocked` as a utility function so we can use it elsewhere.
https://github.com/superdesk/superdesk-client-core/blob/sams/scripts/extensions/sams/src/ui/list/ListItemBorder.tsx#L5